src/views/UserSearch/UserSearch.js

```diff
@@ -434,11 +437,27 @@ class UserSearch extends React.Component {
     }, []);
 
     try {
+      this.context.sendCallout({ message: <FormattedMessage id="ui-users.reports.inProgress" /> });
```

Comment:
Should this line be replaced to line 451? Now we will see both callouts at the same time? Is it the purpose?

Reply:
1. After the "Save & close" button is pressed display the green message "Export in progress" (we need notify that we start generate report)
2. Get data for report (it can take some time)
3. If no data is found display the red message "No items found." or start generate report and after generating the report, add the ability to download it.

This will be consistency with "Claimed returned report (CSV)"